code/modules/unit_tests/font_awesome_icons.dm

```diff
@@ -0,0 +1,96 @@
+/**
+ * This unit test verifies that all Font Awesome icons are present in code, and that all quirk icons are valid.
+ */
+/datum/unit_test/font_awesome_icons
+	var/font_awesome_css
+	var/list/allowed_icons
+
+/datum/unit_test/font_awesome_icons/Run()
+	var/font_awesome_file = file('html/font-awesome/css/all.min.css')
+	if(isnull(font_awesome_file))
+		TEST_FAIL("Font Awesome CSS file could not be found!")
+		return
+
+	font_awesome_css = file2text(font_awesome_file)
```

Comment:
This can probably be condensed down right?

Reply:
I believe file2text throws an exception if you try to pass in null